src/pc/protocols/usb_host.cpp

```diff
@@ -42,6 +51,7 @@ static constexpr int XLINK_USB_DATA_TIMEOUT = 0;
 static unsigned int bulk_chunklen = DEFAULT_CHUNKSZ;
 static int write_timeout = DEFAULT_WRITE_TIMEOUT;
 static int initialized;
+static bool isServer;
```

Comment:
make this an atomic and initialize to false just to make sure we are not potentially reading it uninitialized in some codepaths

```suggestion
static std::atomic<bool> isServer{false};
```